nlp/nl2query/NL2Query_showcase.ipynb

```diff
@@ -0,0 +1,304 @@
+{
```

Comment:
@ahandan @matprov 
I wonder if we should move this notebook in the [notebook folder](https://github.com/crim-ca/pavics-jupyter-images/tree/master/nlp/notebooks)? This notebook will just be copied on the image into the nl2query folder right now, so it won't be accessible by a user. 
I'm guessing this notebook might be intended to be available on the tutorial-notebooks folder on the JupyterLab environment?

Reply:
yes, good idea!